syntax = "proto3";

option go_package = "github.com/souzjfe/grpc-golang-python/proto";

package number;

service Number {
    rpc SendNumber(NumberRequest) returns (MaxNumberResponse) {}
}

message NumberRequest {
    int32 number = 1;
}

message MaxNumberResponse {
    int32 max_number = 1;
}
